stats/stats/src/blockscout_indexing.rs

```diff
@@ -16,12 +16,15 @@ pub async fn is_blockscout_indexing(
         min_block_saved = min_block_saved,
         "checking min block in blockscout database"
     );
-    if min_block_blockscout < min_block_saved {
-        set_min_block_saved(db, min_block_blockscout).await?;
-        Ok(true)
-    } else {
-        Ok(false)
-    }
+    Ok(min_block_blockscout < min_block_saved)
+}
+
+pub async fn save_indexing_info(
+    blockscout: &DatabaseConnection,
+    db: &DatabaseConnection,
+) -> Result<(), DbErr> {
+    let min_block_blockscout = get_min_block_blockscout(blockscout).await?;
```

Comment:
that's not ideal too
we should write here the value we've read

or:
1. we've read 1000 as min blockscout block
2. while we were updating, blockscout indexed also 500-1000 blocks. some of our updates updated that blocks, some don't
3. we've written 500 as min block, and then blockscout stopped indexing
4. some of the charts don't have 500-1000 blocks updated